README.md

```diff
@@ -175,9 +175,10 @@ Different platforms enable slightly different experiences, but generally finds
 a real-world surface and allows the user to place the model, to be viewed through
```

Comment:
in model viewer without tap xr its possible automatically ar is open

Reply:
No, you can't enter AR mode without the user tapping. Browsers disallow this for security.